README.md

```diff
@@ -46,18 +51,30 @@ Download and install [Ubuntu 24.04 server](https://releases.ubuntu.com/24.04/ubu
 
 ### 4.2 Enable Intel TDX in Host OS
 
-1. Download this repository by downloading an asset file from the [releases page on GitHub](https://github.com/canonical/tdx/releases) or by cloning the repository (at the appropriate tag/branch).
+1. Download this repository by downloading an asset file from the [releases page on GitHub](https://github.com/canonical/tdx/releases) or 
+   by cloning the repository (at the appropriate tag/branch).
+   
+   For example: 
+   ```bash
+   git clone https://github.com/canonical/tdx.git
+   git checkout noble-24.04
+   ```
+
+2. Customize the setup by editing the global configuration file `setup-tdx-config`.  
```

Comment:
This file `setup-tdx-config` is not only for the host setup customization, it contains the customizations for the guest image also;

Reply:
I changed it to "Customize the setup of the host and TD by editing the configuration..." to be more explicit.